src/ptf/mask.py

```diff
@@ -84,14 +83,14 @@ def __str__(self):
 
 
 def utest():
-    p = scapy.Ether() / scapy.IP() / scapy.TCP()
+    p = packet.Ether() / packet.IP() / packet.TCP()
     m = Mask(p)
     assert m.pkt_match(p)
-    p1 = scapy.Ether() / scapy.IP() / scapy.TCP(sport=97)
+    p1 = packet.Ether() / packet.IP() / packet.TCP(sport=97)
     assert not m.pkt_match(p1)
-    m.set_do_not_care_scapy(scapy.TCP, "sport")
```

Comment:
What do you think or renaming also this method into `set_do_not_care_packet` ??? I left it untouched since it can be painful to change this name in all our tests...

Reply:
We can always create method with new name invoke old there and deprecate old one, this way we don't have to change it at once and everything will still work